src/lib/hooks.js

```diff
@@ -42,6 +42,10 @@ const api = {
 		CustomFields.setCustomField(key, value, overwrite);
 	},
 
+	setCustomTitle(customTitle) {
+		store.setState({ customTitle });
+	},
+
 	setTheme({ color, fontColor, iconColor } = {}) {
```

Comment:
Let's use the `setTheme` method to add the new parameter, `title`.

Reply:
That's better implementation (y)